resources/js/visual-editor/editor/visual-editor-theme.css

```
@@ -0,0 +1,77 @@
+        sans-serif
+    );
+}
+
+/**
+ * Scope Gutenberg's serif block canvas typography inside the editor shell
+ * so block content retains the expected editorial feel without bleeding
+ * the host admin's sans-serif into block headings and paragraphs.
+ */
+.ap-visual-editor__shell .editor-styles-wrapper {
+    --wp-admin-theme-color: var(--color-primary, #2563eb);
+    color: var(--color-base-content, #1f2937);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)